steps/bitrise-step-qyrus-mobility-test-runner-android/0.0.1/step.yml

```diff
@@ -0,0 +1,125 @@
+title: qyrus-mobility-test-runner-android
+summary: Upload apk and ipa to Qyrus platform, triggers the tests.
+description: |
+  This step uploads the apk or ipa to Qyrus test platform, runs the test on real devices.
+website: https://github.com/qyrusai/bitrise-step-qyrus-mobility
+source_code_url: https://github.com/qyrusai/bitrise-step-qyrus-mobility
+support_url: https://github.com/qyrusai/bitrise-step-qyrus-mobility/issues
+published_at: 2021-11-19T23:04:59.213238+05:30
+source:
+  git: https://github.com/QyrusAI/bitrise-step-qyrus-mobility-test-runner-android.git
+  commit: d20189f0b554641351dccbf648ee6b4bb2419883
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: node
+  apt_get:
+  - name: nodejs
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- gateway_url: null
+  opts:
+    description: |
+      Enter the url of the gateway which is provided by Qyrus else
+      contact support@qyrus.com
+    is_required: true
+    summary: Qyrus provided gateway url
+    title: gateway_url
```

Comment:
The `title` field of an input should be human-readable, I suggest `Gateway URL`. This is true for other inputs as well